examples/docs_beta_snippets/docs_beta_snippets/guides/data-assets/quality-testing/freshness-checks/external-asset-freshness-check.py

```diff
@@ -0,0 +1,55 @@
+from datetime import timedelta
+
+import dagster_snowflake as dg_snowflake
+
+import dagster as dg
+
+
+@dg.observable_source_asset(specs=[dg.AssetSpec("hourly_sales")])
+def hourly_sales(snowflake: dg_snowflake.SnowflakeResource):
+    with snowflake.get_connection() as conn:
+        freshness_results = dg_snowflake.fetch_last_updated_timestamps(
+            snowflake_connection=conn.cursor(),
+            tables=["hourly_sales"],
+            schema="PUBLIC",
+        )
+        for table_name, last_updated in freshness_results.items():
+            yield dg.ObserveResult(
+                asset_key=table_name,
+                metadata={
+                    "dagster/last_updated_timestamp": dg.MetadataValue.timestamp(
```

Comment:
this metadata is pretty magical it might be worth adding a brief description of it in the docs, something to the effect of the metadata is a special key used by freshness checks

Reply:
+1

Reply:
I wonder if we should add a list of dagster metadata keys in the docs somewhere, along with what the purpose of each key. Currently these special keys are referenced in scattered places ([for example](https://docs.dagster.io/concepts/metadata-tags/asset-metadata/column-level-lineage#for-assets-defined-in-dagster)) which makes them hard to search for. 

And ideally when you're adding metadata, you can look at a single list and see all the different things you can emit.

Reply:
We currently have this: https://docs.dagster.io/concepts/metadata-tags/asset-metadata#standard-asset-metadata-entries

Btw, just noticed that docs search for "metadata" only returns API reference links.

Reply:
We should definitely have a metadata reference page. I think just mentioning that this key is what the freshness check uses should be sufficient, and we can add a link to the metadata page once created.

Reply:
+1 on the metadata reference.

Reply:
Updated to mention the key and added a TODO for adding it to the metadata reference